x-pack/plugins/reporting/server/lib/level_logger.ts

```diff
@@ -5,14 +5,14 @@
  * 2.0.
  */
 
-import { LoggerFactory } from 'src/core/server';
+import { LoggerFactory, LogMeta } from 'src/core/server';
```

Comment:
This file will be removed, hopefully soon: https://github.com/elastic/kibana/issues/125011